frontend/src/state/GraphState.jsx

```diff
@@ -427,79 +428,86 @@ export class GraphContainer {
     let displayOrder = 0;
     let rows = {
       displayedRows: {
-        name: "Visible rows", displayOrder: displayOrder++,
+        name: "Visible rows",
+        displayOrder: displayOrder++,
         value: displayedConcepts.length,
       },
       concepts: {
-        name: "Concepts", displayOrder: displayOrder++,
-        value: this.gd.concept_ids.length,
-        hiddenConceptCnt: setOp('difference', this.gd.concept_ids, displayedCids).length,
-        displayedConceptCnt: setOp('intersection', this.gd.concept_ids, displayedCids).length,
+        name: "Concepts",
+        displayOrder: displayOrder++,
+        value: this.gd.concept_ids?.length ?? 0,
+        hiddenConceptCnt: setOp('difference', this.gd.concept_ids ?? [], displayedCids).length,
```

Comment:
## Edge case: Potentially undefined `concept_ids`
I [asked ChatGPT](https://chatgpt.com/share/64697079-8e8f-460d-8006-ee4040341645) to handle cases of potentially undefined `PROPERTY` in `this.gd.specialConcepts.PROPERTY.length?` patterns, but it was overzealous and handled other situations as well.
@Sigfried I leave it up to you if you want to keep this change or revert.